Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -258,7 +258,12 @@ void Wiimote::HandleRequestStatus(const OutputReportRequestStatus&)
 
 void Wiimote::HandleWriteData(const OutputReportWriteData& wd)
 {
-  // TODO: Are writes ignored during an active read request?
+  if (m_read_request.size)
+  {
+    // FYI: Writes during an active read will occasionally produce a "busy" (0x4) ack.
+    // We won't simulate that as it often does work. Poorly programmed games may rely on it.
```

Comment:
Should this be "doesn't" ?

Reply:
No. "it often does work." which is what is simulated here.

Reply:
Hmm ok.  So you're saying we aren't sending a busy ack like for reads because on console it is so sporadic and a write will work anyway most of the time?

Reply:
Yes. It the write will work "most of the time" on real hardware.

Reply:
Perhaps "We won't simulate that as those reads still do often work, and poorly programmed games may rely on it despite it occasionally failing on hardware"?

That said, I don't think there are that many games that write to EEPROM, and I think the reading/writing is handled in Nintendo's SDK and that includes reading it back after writing to check, so it *probably* is fine to be strict.  I doubt a poorly programed game rolled its own EEPROM handling functions, but who knows...